modules/widgets/src/lib/geocode/geocoder-history.ts

```diff
@@ -0,0 +1,72 @@
+// deck.gl
+// SPDX-License-Identifier: MIT
+// Copyright (c) vis.gl contributors
+
+import {type Geocoder} from './geocoder';
+
+const CURRENT_LOCATION = 'current';
+const LOCAL_STORAGE_KEY = 'deck-geocoder-history';
+
+export type GeocoderHistoryProps = {};
+
+/**
+ * A widget that display a text box that lets user type in a location
+ * and a button that moves the view to that location.
+ * @todo For now only supports coordinates, Could be extended with location service integrations.
+ */
+export class GeocoderHistory {
```

Comment:
```suggestion
export class GeocoderStore {
```

The name is a bit confusing since the class is also responsible to UI state, not just the history.